From 25dd4944996e746f1aee73bcbba4889f48e7ef9f Mon Sep 17 00:00:00 2001
From: Lawrence Rust <lvr@softsystem.co.uk>
Date: Mon, 7 Jul 2014 18:04:49 +0100
Subject: [PATCH 09/56] MythUIWebBrowser: silence a warning when we can't find
 the parent window

Base widgets don't necessarily have a window as a parent so don't log
an error in this case.

Signed-off-by: Paul Harrison <pharrison@mythtv.org>
---
 mythtv/libs/libmythui/mythuiwebbrowser.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythui/mythuiwebbrowser.cpp b/mythtv/libs/libmythui/mythuiwebbrowser.cpp
index 2db65a7..3c16815 100644
--- a/mythtv/libs/libmythui/mythuiwebbrowser.cpp
+++ b/mythtv/libs/libmythui/mythuiwebbrowser.cpp
@@ -957,7 +957,7 @@ void MythUIWebBrowser::Init(void)
         parentObject = parentObject->parent();
     }
 
-    if (!m_parentScreen)
+    if (!m_parentScreen && parent() != GetGlobalObjectStore())
         LOG(VB_GENERAL, LOG_ERR,
             "MythUIWebBrowser: failed to find our parent screen");
 
-- 
1.7.10.2

